Automattic/portkey-android
# Tap to delete slide

Closes #423.

Deleting a slide is now done by tapping the current slide in the frame selector, and I removed the overflow menu we used to keep this functionality in.

![tap-to-delete-slide](https://user-images.githubusercontent.com/9613966/95557023-2d155900-0a4f-11eb-8c7e-c05510a0b6ff.gif)

### To test:
1. Tapping a slide should hide most of the UI and show the delete button
2. Tapping on another slide, or the same slide again, dismisses the delete button
3. Tapping anywhere else on the UI should dismiss the delete button and handle the tap (e.g. tapping on an added view will open the editor)
4. Adding a new slide while the delete button is visible should dismiss the delete button
